src/Compilers/CSharp/Test/Emit/CodeGen/SwitchTests.cs

```diff
@@ -4186,7 +4186,99 @@ .maxstack  2
         [Fact()]
         public void IsWarningSwitchEmit()
```

Comment:
This doesn't have much utility beyond comparing the result. Let me know if I should revert since large codegen tests are generally avoided.